File: DKS-API/Models/DKS/DevBomFile.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DKS.API.Models.DKS
{
    public class DevBomFile
    {
        [Key]
        public string FACTORY { get; set; }

        public string DEVTEAMID { get; set; }

        public string SEASON { get; set; }

        public string MODELNO { get; set; }

        public string MODELNAME { get; set; }

        [Key]
        public string ARTICLE { get; set; }
        [Key]
        public string STAGE { get; set; }

        public short VER { get; set; }

        [Key]
        public short SORT { get; set; }

        public string FILENAME { get; set; }

        public string REMARK { get; set; }

        public string APPLY { get; set; }

        public string ECRNO { get; set; }

        public DateTime? UPDAY { get; set; }

        public string UPUSR { get; set; }
        public string PDM_APPLY { get; set; }
        public string PDM_UPUSR { get; set; }
        public DateTime? PDM_UPDAY { get; set; }
        public string ARTICLE_LIST { get; set; }
        
        
    }
}

File: DKS-SPA/src/app/core/_models/dev-bom-file.ts
import { ModelInterface } from "./interface/model-interface";

export class DevBomFile implements ModelInterface {
  factory: string;
  devTeamId: string;
  season: string;
  modelNo: string;
  modelName: string;
  article: string;
  stage: string;
  ver: number;
  sort:number;  //0817
  fileName: string;
  remark: string ;
  apply: string;
  ecrno: string;  //0817
  upDay: Date ;
  upUsr: string;
  pdmApply: string;  //0817
  pdmUpusr: string;  //0817
  pdmUpday: Date;  //0817 1216
  
  articleList: string;  //1216
  
  constructor(){
    
    this.ver = 0;

  }
}
